modules/juce_audio_devices/native/java/app/com/rmsl/juce/JuceMidiSupport.java

```diff
@@ -485,12 +485,16 @@ public void removePort (MidiPortPath path)
             openPorts.remove (path);
         }
 
+        @Override
         public void onDeviceAdded (MidiDeviceInfo info)
         {
-            manager.openDevice (info, this, null);
+            if (info.getType() != MidiDeviceInfo.TYPE_BLUETOOTH) {

```

Comment:
This isn't called on all phones, but it turns out that when it is, Bluetooth MIDI devices need separate handling already done in BluetoothMidiPairing.